notebooks/deconvo_sanity_checks.py

```diff
@@ -28,10 +28,12 @@
 signature_almudena = read_almudena_signature(
     "/home/owkin/project/Almudena/Output/Crosstiss_Immune_norm/CTI.txt"
 )  # it is the normalised one (using adata.X and not adata.raw.X, to match this code)
+# TO ADD : the new signature with granular updated cell types
 
 
 # %%
 # cell type categories
+# TO ADD : possibility of crosstissue_granular_updated
```

Comment:
will be added in another PR ?

Reply:
Yes! I guess it can be done in the same one as when integrating MixUpVI to the sanity checks